docs/DeveloperGuide.md

```diff
@@ -290,28 +290,32 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+
+**Use case: UC2 - List all people saved in `BizBook`**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to list all people saved in their `BizBook`
+2.  `BizBook` shows a list of persons
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. If there are no contacts stored in the system.
+    * 1a1 `BizBook`shows an error message.
+
+      Use case ends.
```

Comment:
Should empty list be invalid? What if the contact list is just initialized/cleared?

Reply:
I think so? 

I was just following our feature list, where we treat a newly initialized/cleared list as an invalid input then we'll show a message like "You have no contact saved" but i generalized it to just be error message